app/blueprints/rdv_plan_blueprint.rb

```diff
@@ -8,11 +8,17 @@ class RdvPlanBlueprint < Blueprinter::Base
   end
 
   field :rdv do |rdv_plan, _options|
-    next if rdv_plan.rdv.nil?
+    # L'appli partenaire a besoin des infos de base sur le rendez-vous pour permettre à l'agent
+    # de savoir quand et comment il aura lieu.
+    rdv = rdv_plan.rdv
+
+    next if rdv.nil?
 
     {
-      id: rdv_plan.rdv_id,
-      status: rdv_plan.rdv.status,
+      id: rdv.id,
+      status: rdv.status,
+      starts_at: rdv.starts_at.iso8601,
```

Comment:
Je suis un peu frileux à l'idée d'avoir plusieurs formats de datetime dans notre API : si j'utilise une API qui me renvoie des formats de datetime instables, je vais avoir des envies de meurtre.

Je suggère qu'on active le format `iso8601` sur l'ensemble de l'API, à travers la config donnée dans la section "Custom Formatting for Dates and Times" du [README de `blueprinter`](https://github.com/procore-oss/blueprinter).

Bien sûr ça peut se faire dans une PR séparée.

Et il est possible qu'à un moment on soit amenés à avoir aussi les champs `starts_at_utc` et `starts_at_tz`, mais c'est du moyen / long terme. :wink: :clock1:

Reply:
Oui, je pense que tu as raison. Là ça me permettait d'aller vite (surtout pour la spec), et il me semble que les différents formats de date qu'on a dans l'api sont tous iso8601 compatibles, mais juste légèrement différents. Mais c'est peut-être plus malin de ne pas faire de custom pour cet endpoint, et plutôt d'ajuster la spec.
Je vais faire ça en début de semaine prochaine, et merger après.

Reply:
c'est updaté